tests/tests

```diff
@@ -89,6 +89,20 @@
   input = 'HourlyObjectOrientedTest.py'
  [../]
 
+ [./CashFlow_PyomoObjectOriented]
+  type = 'RavenPython'
+  input = 'HourlyObjectOrientedTestWithPyomoExpression.py'
+  python3_only = True
+  required_libraries = 'pyomo'
+ [../]
```

Comment:
I think this PyomoObjectOriented is superseded by the PyomoTest; also, I think it can be named just PyomoTest instead of CashFlow_PyomoTest